readme.md

```diff
@@ -0,0 +1,70 @@
+Misc-deps is a collection of various static dependencies used by [NVDA](https://github.com/nvaccess/nvda).
+
+# Updating contents
+
+## include
+
+### Adobe Acrobat Access
+
+1. Download the latest [Adobe Acrobat SDK](https://developer.adobe.com/console/servicesandapis)
+1. Extract the zip
+1. Copy the file from `Adobe\Acrobat DC SDK\Version 1\AccessibilitySupport\AcrobatAccess.idl`
+
+### iSimpleDOM
+
+MSAA interfaces for gecko / firefox.
+
+Update from [here](https://github.com/mozilla/gecko-dev/tree/master/accessible/interfaces/msaa)
+
+### mathPlayer
+
+Project has been abandoned, updates are unlikely.
+In NVDA this will be abandoned soon in favour of MathCAT.
+
+## python
+Extract python3.dll from your python install path.

```

Comment:
Shouldn't py2exe do this?

Reply:
Could you elaborate how? I couldn't find a method to do this built in to py2exe

Reply:
I was thinking about adding an entry to the data_files list, but not sure whether that'll work.

Reply:
It very likely would, as we use the same way to copy additional dll's for BrlAPI. Having said that it is also quite possible that, as soon as the new build of BrlAPI which requires this library will be added into miscdeps, py2exe would be smart enough to include the dll on its own.

Reply:
>I was thinking about adding an entry to the data_files list, but not sure whether that'll work.

But where would it pull from? python3.dll isn't in our virtual environment, so it would have to pull from the python path, which may not be standard for all dev environments outside of AppVeyor.
We will have to see if BrlAPI pulls this in automatically once BRLTTY correctly uses the stable ABI.

Reply:
> But where would it pull from?

When virtual environment is active `sys.base_prefix` points to the Python install path. As far as I can tell `python3.dll` is always placed in the Python install folder.

Reply:
Thanks, should have checked for this